cpp/arcticdb/storage/test/test_storage_factory.cpp

```diff
@@ -1,73 +0,0 @@
-/* Copyright 2023 Man Group Operations Limited
- *
- * Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
- *
- * As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
- */
-
-#include <gtest/gtest.h>
-
-#include <arcticdb/codec/codec.hpp>
-#include <arcticdb/storage/storage_factory.hpp>
-#include <arcticdb/storage/library.hpp>
-#include <arcticdb/storage/library_index.hpp>
-#include <arcticdb/storage/config_resolvers.hpp>
-#include <arcticdb/util/pb_util.hpp>
-#include <arcticdb/util/test/config_common.hpp>
-#include "folly/Range.h"
-
-#include <unordered_map>
-#include <vector>
-#include <type_traits>
-
-namespace ac = arcticdb;
-namespace as = arcticdb::storage;
-namespace asl = arcticdb::storage::lmdb;
-
-TEST(TestStorageFactory, LmdbLookup) {
```

Comment:
Deleting tests doesn't look good. Are they moved somewhere? Or this file was forgotten in the CMake?